.travis.yml

```diff
@@ -3,45 +3,48 @@ notifications:
     - oncall-dfusion@gnosis.io
   if: (branch = master) OR (tag IS present)
 if: (branch = master) OR (type = pull_request) OR (tag IS present)
+
+os: linux
+dist: xenial
 language: rust
-rust: stable
-env:
-  global:
-    - CARGO_INCREMENTAL=0
-matrix:
+
+install:
+  - rm -rf "/home/travis/.cache/node-gyp/"
+  - nvm install 10 && nvm alias default 10 && npm install -g yarn@latest && yarn --version
+  - (cd dex-contracts && yarn --frozen-lockfile && yarn run prepack)
+  - ./scripts/setup_contracts.sh
+
+jobs:
+  fast_finish: true
+  allow_failures:
+    - rust: nightly
+
   include:
-    - stage: "Local and Rinkeby in Parallel"
-      name: "Build and Local Testing"
+    - name: "Build and Local Testing"
+      rust: stable
       cache:
         directories:
           - $HOME/.cache/node-gyp
           - $HOME/.cache/yarn
           - $HOME/.cargo/bin
       before_cache:
         - rm -rf "$TRAVIS_HOME/.cargo/registry/src"
-      before_install:
-        - sudo apt-get update && sudo apt-get install -y python-pip && sudo pip install awscli
-        - $(aws ecr get-login --no-include-email --region $AWS_REGION)
```

Comment:
Is this no longer needed to pull the docker image?

Reply:
Oh that is why it is needed. I only just readded the old tests so I didn't yet notice CI failing because it couldnt' pull.